### PR TITLE
Fix something I forgot

### PR DESCRIPTION
**Pull request by larziwau**
Fix something I forgot in the build ci. I'm unsure if i should upgrade sccache though
Also update Python

### CODE REVIEW

**Comment by larziwau** [comment]
tried a fix for the update submodule thingy

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -627,7 +627,7 @@ jobs:
       - name: Setup Git
         run: |
           git config --global user.name "github-actions[bot]"
-          git config --global user.email "github-actions[bot]@users.noreply.github.com"
+          git config --global user.email "41898282+github-actions[bot]@users.noreply.github.com"
```
Comment (mr-cheffy) [inline .github/workflows/build.yml]: What's this for?

**Review by mr-cheffy — CHANGES_REQUESTED** [review]

File: .github/workflows/build.yml (hunk 1 above)
Comment (larziwau) [reply .github/workflows/build.yml]: well its documented in the actions/checkout repo, this is the official email of actions bot

Comment (chenasraf) [reply .github/workflows/build.yml]: I was curious so I looked it up, it's correct, that's the proper attached user id for github actions bot,

See:
- https://github.com/actions/checkout/pull/1707 - an official merged PR including this info
- https://api.github.com/users/github-actions%5Bbot%5D - the API user info request from GitHub by username

Comment (larziwau) [reply .github/workflows/build.yml]: yes, this is why i made this pr

**Comment by larziwau** [comment]
anyways is there anything about the pr?

**Pull request closed without merging**